Dockerfile

```diff
@@ -0,0 +1,63 @@
+FROM ubuntu:14.04
+MAINTAINER Cezary Baginski <cezary@chronomantic.net>
+
+# Packages needed to install RVM and run Bundler gem commands
+RUN apt-get update -qq && apt-get -y install ca-certificates curl git-core --no-install-recommends && rm -rf /var/lib/apt/lists/* /var/cache/apt/archives/*.deb /var/cache/apt/archives/partial/*.deb /var/cache/apt/*.bin
+
+# Create guest user early (before rvm) so uid:gid are 1000:000
+RUN useradd -m -s /bin/bash guest
+
+# Temporarily install RVM as root - just for requirements
+RUN gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3
+RUN curl -L get.rvm.io | bash -s stable
+RUN bash -l -c 'rvm requirements 2.2.1'
+RUN bash -l -c 'echo yes | rvm implode'
+
+# Fix locale
+ENV DEBIAN_FRONTEND noninteractive
+RUN dpkg-reconfigure locales && locale-gen en_US.UTF-8 && /usr/sbin/update-locale LANG=en_US.UTF-8
+RUN echo 'en_US.UTF-8 UTF-8' >> /etc/locale.gen && locale-gen
+ENV LC_ALL C.UTF-8
+ENV LANG C.UTF-8
+ENV LANGUAGE C.UTF-8
+
+USER guest
+ENV HOME /home/guest
+WORKDIR /home/guest
+
+# Install RVM as guest
+RUN gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3
+RUN /bin/bash -l -c "echo 'gem: --no-ri --no-rdoc' > ~/.gemrc"
+RUN curl -L get.rvm.io | bash -s stable
+RUN /bin/bash -l -c "rvm install 2.3.0"
+RUN /bin/bash -l -c "gem install bundler --no-ri --no-rdoc"
```

Comment:
> I really want to add jruby etc. 

If you decide to put JRuby into the image, you're automatically forcing EVERYONE to have it.

RVM is a biggest convenience here. It's trivial to then have ANY Ruby at ANY patchlevel.

And installing any Ruby is usually a matter of minutes, so this isn't a problem. A workflow with Docker Compose means you can just change a line in the Dockerfile, rerun the existing `script/run_tests_in_docker` and everything happens automatically. You don't even need to learn docker. Run the script, everything works. Need to change to JRuby? Just change the line in the Dockerfile and rerun the tests. 

I'd say the image size isn't as much of a nuisance as being forced to wait a couple of minutes and download ALSO JRuby (which many may not intend to even want at all) on a slow network link.

My assumption is: if Ruby 2.3.0 is ready to go, that's all you need to contribute - travis can take care of the rest. If you personally need JRuby for fixing JRuby support, that's "exotic" enough for you to "pay the price" of changing a line of code and rebuilding the code for YOUR convenience.

Installing multiple Rubies by default is a bit like installing multiple editors like Vim and Emacs (and whatever) "just in case someone needs them there too".

That's why I "stop" at RVM + Ruby 2.3.0. It's "enough" to run the tests with enough confidence that they'll work on Travis. Everything else is up to the contributor to decide. They can install KDE there if the want ... but I wouldn't make that the default.

(For 99% of the contributors out there, it isn't a game changer to have JRuby preinstalled).